### ISSUE TITLE
no output data returned

### ISSUE THREAD

**Issue by dynahpointch** [umanamente/n8n-nodes-imap#36]
Hello, thank for your work the node works well, but I've run into a problem.

I use the Mail Server Xeams (https://www.xeams.com/home.htm) and it can fetch all folders to show as selection in the mailbox "From List".
It will receive all Mails (log file from xeams) but does not generate any output data in n8n.
If i use an imap account on a exchange server it works.

Tested with 2.3.0 / 2.4.1

I have only the Log from Xeams.

```
2024-11-14 15:26:09,243 -  - [      13] ************** New Connection from: 192.168.0.5:143 ************** 
2024-11-14 15:26:09,244 -  - [      13] S <-- * OK Synametrics IMAP4rev1 server ready 11/14/24 3:26 PM
2024-11-14 15:26:09,258 -  - [      13] C --> 1 CAPABILITY
2024-11-14 15:26:09,259 -  - [      13] S <-- * CAPABILITY IMAP4rev1 IDLE
2024-11-14 15:26:09,259 -  - [      13] S <-- 1 OK CAPABILITY completed
2024-11-14 15:26:09,306 -  - [      13] C --> 2 LOGIN "test@mailaccount.com" xxxxxx
2024-11-14 15:26:09,307 -  - [      13] S <-- 2 OK LOGIN completed
2024-11-14 15:26:09,329 -  - [      13] C --> 3 CAPABILITY
2024-11-14 15:26:09,329 -  - [      13] S <-- * CAPABILITY IMAP4rev1 IDLE
2024-11-14 15:26:09,329 -  - [      13] S <-- 3 OK CAPABILITY completed
2024-11-14 15:26:09,365 -  - [      13] C --> 4 LIST "" ""
2024-11-14 15:26:09,365 -  - [      13] S <-- * LIST (\Noselect) "/" ""
2024-11-14 15:26:09,365 -  - [      13] S <-- 4 OK LIST completed
2024-11-14 15:26:09,397 -  - [      13] C --> 5 LIST "" "INBOX"
2024-11-14 15:26:09,400 -  - [      13] S <-- * LIST () "/" "INBOX"
2024-11-14 15:26:09,400 -  - [      13] S <-- 5 OK LIST completed
2024-11-14 15:26:09,424 -  - [      13] C --> 6 LSUB "" "INBOX"
2024-11-14 15:26:09,425 -  - [      13] S <-- * LSUB () "/" "INBOX"
2024-11-14 15:26:09,425 -  - [      13] S <-- 6 OK LIST completed
2024-11-14 15:26:09,436 -  - [      13] C --> 7 EXAMINE INBOX
2024-11-14 15:26:09,454 -  - [      13] S <-- * FLAGS (\Seen \Deleted)
2024-11-14 15:26:09,454 -  - [      13] S <-- * 2 EXISTS
2024-11-14 15:26:09,454 -  - [      13] S <-- * OK [UIDNEXT 3] Predicted next UID
2024-11-14 15:26:09,454 -  - [      13] S <-- * 0 RECENT
2024-11-14 15:26:09,454 -  - [      13] S <-- * OK [PERMANENTFLAGS (\Deleted \Seen)]
2024-11-14 15:26:09,454 -  - [      13] S <-- 7 OK [READ-ONLY] SELECT completed
2024-11-14 15:26:09,496 -  - [      13] C --> 8 UID SEARCH ALL
2024-11-14 15:26:09,499 -  - [      13] S <-- * SEARCH 1 2
2024-11-14 15:26:09,506 -  - [      13] S <-- 8 OK UID completed
2024-11-14 15:26:09,517 -  - [      13] C --> 9 UID FETCH 1:2 (UID ENVELOPE)
2024-11-14 15:26:09,789 -  - [      13] S <-- * 1 FETCH (UID 1 ENVELOPE ("Thu, 14 Nov 2024 12:30:02 +0000" "<SUBJECT>" (("hello@world.com" NIL "hello" "world.com")) (("hello@world.com" NIL "hello" "world.com")) (("hello@world.com" NIL "hello" "world.com")) (("helloFROM@worldfrom.com" NIL "helloFROM" "worldfrom.com")) NIL NIL NIL "<ZR0P278MB02686EC0E225A0AAA9DB847FC95B2@ZR0P278MB0268.CHEP278.PROD.OUTLOOK.COM>"))
2024-11-14 15:26:09,812 -  - [      13] S <-- * 2 FETCH (UID 2 ENVELOPE ("Thu, 14 Nov 2024 12:04:05 +0000" "test" (("hello@world.com" NIL "hello" "world.com")) (("hello@world.com" NIL "hello" "world.com")) (("hello@world.com" NIL "hello" "world.com")) (("helloFROM@worldfrom.com" NIL "helloFROM" "worldfrom.com")) NIL NIL NIL "<ZR0P278MB0268AC40E3BF0386AED3C53CC95B2@ZR0P278MB0268.CHEP278.PROD.OUTLOOK.COM>"))
2024-11-14 15:26:09,812 -  - [      13] S <-- 9 OK UID FETCH completed
2024-11-14 15:26:09,864 -  - [      13] C --> A LOGOUT
2024-11-14 15:26:09,864 -  - [      13] S <-- * BYE IMAP4rev1 IDLE Server logging out
2024-11-14 15:26:09,864 -  - [      13] S <-- A OK LOGOUT completed
2024-11-14 15:26:09,865 -  - [      13] ~~~~~~~~~~~~~~ Connection closed: 192.168.0.5 (622 ms)~~~~~~~~~~~~~~ 

```
i set up a test environment in the oracle cloud you can try it yourself
144.24.250.201
hello@world.com
H8ll0W0rld
IMAP Port 143

**Comment by dynahpointch** [comment]
![image](https://github.com/user-attachments/assets/fac8e72c-30ff-4f04-af67-80132c77a260)

**Comment by dynahpointch** [comment]
![image](https://github.com/user-attachments/assets/ecc462d0-5617-4e77-92a2-a69ba57c58e6)

**Comment by dynahpointch** [comment]
![image](https://github.com/user-attachments/assets/596ca96c-8b06-4c27-966a-49c10ffa92b7)

**Comment by umanamente** [comment]
I reproduced the issue; it seems to be related to [ImapFlow](https://github.com/postalsys/imapflow).

I can create an [issue in the ImapFlow repository](https://github.com/postalsys/imapflow/issues). May I include your logs and test credentials?

**Comment by dynahpointch** [comment]
Yes, this setup is specifically designated as a test environment created exclusively for addressing this particular issue.

**Comment by umanamente** [comment]
Created https://github.com/postalsys/imapflow/issues/237

**Comment by umanamente** [comment]
The issue has been resolved in version 2.4.2. 
If possible, please keep the test server so that ImapFlow developers can investigate the issue. (https://github.com/postalsys/imapflow/issues/237).

**Comment by umanamente** [comment]
Xeams support team confirmed that this is a bug and informed me that it will be fixed in future updates.
